gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/test/java/io/gravitee/gateway/standalone/http/LoggableClientResponseWithRequestTimeoutTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.standalone.http;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import io.gravitee.common.http.HttpStatusCode;
+import io.gravitee.gateway.standalone.junit.annotation.ApiDescriptor;
+import io.gravitee.gateway.standalone.reporter.AbstractReporterFakerAndWiremockGatewayTest;
+import io.gravitee.gateway.standalone.reporter.FakeReporter;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Florent CHAMFROY (florent.chamfroy at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@ApiDescriptor(value = "/io/gravitee/gateway/standalone/http/teams.json", configFolder = "/gravitee-requestTimeout")
+public class LoggableClientResponseWithRequestTimeoutTest extends AbstractReporterFakerAndWiremockGatewayTest {
+
+    private boolean hasFakerReported;
+
+    protected final CountDownLatch latch = new CountDownLatch(1);
+
+    @Override
+    public void addFakeReporterHandler(FakeReporter fakeReporter) {
+        fakeReporter.setReportableHandler(
+            reportable -> {
```

Comment:
You have to check that the reportable is the type you are looking for (metrics or log) and not something else (healthcheck, monitoring)